inventory/sample.casl.rht-labs.com.d/files/ipa-ca.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I'd recommend renaming this directory from 'rht-labs.com' to something with 'example.com' and maybe clobber the content of some of the important files ...